src/App.vue

```diff
@@ -97,7 +101,20 @@ export default {
       }
     },
     async changePageCourseList(page) {
-      this.hold = await CoursesDataService.getPage(page - 1);
+      console.log(
```

Comment:
I would go through and remove your console.log statements now that things are working, will make it look better.

Reply:
On it!